feconf.py

```diff
@@ -566,6 +571,7 @@ def get_empty_ratings():
 COLLECTION_DATA_URL_PREFIX = '/collection_handler/data'
 COLLECTION_SUMMARIES_DATA_URL = '/collectionsummarieshandler/data'
 EDITABLE_COLLECTION_DATA_URL_PREFIX = '/collection_editor_handler/data'
+EDITABLE_TOPIC_DATA_URL_PREFIX = '/topic_editor_handler/data'
```

Comment:
I thought it was supposed to be in alphabetical order. Shall I rename it to something else like TOPIC_DATA_URL_PREFIX or something?

Reply:
It doesn't look like the list is in exact alphabetical order since the next item is COLLECTION_RIGHTS_PREFIX.

But I think your proposal sounds good. It's nice to keep all the related stuff in the same place. Perhaps we should rename the collection constant as well so that the list actually is alphabetical.